word_language_model/requirements.txt

```diff
@@ -1 +1 @@
-torch<2.6
+torch>=2.6
```

Comment:
`torch.serialization.safe_globals` I've used in this PR appeared in torch 2.5. The `torch.load` default to load weights was changed to `True` in 2.6. Since we in any case plan to bump dependency to 2.6 by using `torch.accelerate`, I think we can update the requirement to 2.6 right away.